radis/io/cdsd.py

```diff
@@ -116,7 +117,14 @@
 
 
 def cdsd2df(
-    fname, version="hitemp", count=-1, cache=False, verbose=True, drop_non_numeric=True
+    fname,
+    version="hitemp",
+    count=-1,
+    cache=False,
+    verbose=True,
+    drop_non_numeric=True,
+    wavenum_min=0.0,
```

Comment:
It's not straightforward what wavenum_min and max do here. Can you : 
- change the name to something explicit ? Ex : `load_only_wavenum_above=` ,  `load_only_wavenum_below=`? 
- add the names in the docstrings under Other Parameters with a shortdescription. In the docs, if needed, you can also refer to other functions like load_h5_cache_file using ``` :py:func`~radis.misc.cache_files` ```